scip-java/src/main/scala/com/sourcegraph/scip_java/commands/IndexDependencyCommand.scala

```diff
@@ -24,6 +25,8 @@ final case class IndexDependencyCommand(
     index: IndexCommand = IndexCommand(),
     @Hidden
     snapshotCommand: SnapshotCommand = SnapshotCommand(),
+    @Repeated
+    repositories: Option[List[String]] = None,
```

Comment:
nit: I normally use `List[String]` to avoid having two representations for the empty case (`None` and `Some(Nil)` )